### PR TITLE
Incorporate Schema Migrator

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

Incorporate Schema Migrator from control-plane repository and make it an internal component to discourage external usage.

**Related issue(s)**

See #1185

### CODE REVIEW

**Review by IwonaLanger — APPROVED** [review]

File: Dockerfile.schemamigrator [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+CGO_ENABLED=0 go build -o /build/schema-migrator cmd/schemamigrator/main.go
+
+# Final image
+FROM scratch
+LABEL source=git@github.com:kyma-project/kyma-environment-broker.git
+
+WORKDIR /migrate
+
+COPY --from=build /build/schema-migrator /bin/schema-migrator
+COPY resources/migrations ./migrations
```
Comment (ralikio) [inline Dockerfile.schemamigrator:21]: Shouldn't migration be copied from the `resources/keb/migrations` folder?

**Review by ralikio — DISMISSED** [review]

File: Dockerfile.schemamigrator (hunk 1 above)
Comment (ralikio) [reply Dockerfile.schemamigrator:21]: My mistake.... resources/migrations is correct because we just copy migrations from config map in main.go.

**Review by ralikio — DISMISSED** [review]

**Review by ralikio — APPROVED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**